ui.apps/src/main/content/jcr_root/apps/aem-dictionary-translator/clientlibs/dictionary/js/publish.js

```diff
@@ -49,7 +49,7 @@
         }).done(function () {
             var api = $(collection).adaptTo("foundation-collection");
             if (api && "reload" in api) {
-                api.reload();
+                // api.reload();
```

Comment:
I overlooked this one, on 6.5.17 it's throwing an exception, currently investigating why